Tools/WebKitTestRunner/InjectedBundle/TestRunner.cpp

```diff
@@ -250,34 +250,23 @@ void TestRunner::notifyDone()
     auto& injectedBundle = InjectedBundle::singleton();
     if (!injectedBundle.isTestRunning())
         return;
-
-    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-    bool mainFrameIsRemote = WKBundleFrameIsRemote(WKBundlePageGetMainFrame(injectedBundle.pageRef()));
-    ALLOW_DEPRECATED_DECLARATIONS_END
-    if (mainFrameIsRemote) {
-        setWaitUntilDone(false);
```

Comment:
Can we remove this state from the web process now?  Same with shouldWaitUntilDone